src/components/editor/ComponentPreview.tsx

```diff
@@ -54,6 +54,7 @@ const ComponentPreview: React.FC<{
     case 'ListItem':
     case 'NumberInput':
     case 'BreadcrumbLink':
+    case 'Select':
```

Comment:
Thanks :pray:

Reply:
This should have been detect by the test on this component, we should use the same component list for ComponentPreview and his test.